packages/editor/src/components/post-text-editor/index.js

```diff
@@ -72,8 +81,19 @@ export default function PostTextEditor() {
 		}
 	};
 
+	//Function that checks if html is valid
+	const checkHTML = ( html ) => {
+		const doc = document.createElement( 'div' );
+		doc.innerHTML = html;
+		return doc.innerHTML === html;
+	};
 	return (
 		<>
+			<div className={ isValid ? 'hide' : 'show' }>
```

Comment:
I'm not sure this is the behavior we want... I need to test and think about it.
However, a better a solution to show/hide than classnames is to use the logic with the component like so:
```
{ isValid && <div> ... </div> }
```

That way the markup is not output to the page unless the `isValid` is true.

Reply:
Hi @mkaz  ) I will change my markup , but I have a question for you . It seems parse function on line 36 : 
`
 		const blocks = parse( value );

`

causes a bug that was described in this issue - https://github.com/WordPress/gutenberg/issues/29338 .What is the best behavior to fix this issue  ? It can be a function that removes square brackets before parse function and add them after , or perhaps changing the constant  tokenizer in  gutenberg/packages/block-serialization-default-parser/src/index.js file is better ?